docs/additional.asciidoc

```diff
@@ -2,10 +2,12 @@
 == Additional Information
```

Comment:
Had originally placed this new page under `Additional Information`, but I'm feeling it's not the right fit.

Thoughts on a better section?